### PR TITLE
duplicatesinarray.cpp

### PR DESCRIPTION
**Pull request by Abulaman23**
Thanks for reading my pull request!!

### CODE REVIEW

File: Leetcode/duplicatesinarray.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#include<bits/stdc++.h>
```
Comment (Saikat2407) [inline Leetcode/duplicatesinarray.cpp:1]: first line should be problem link in comments

**Review by Saikat2407 — CHANGES_REQUESTED** [review]
1. add the problem link on top of the file
2. delete the exe file

**Comment by Abulaman23** [comment]
Did the changes,Kindly look into it.

File: Leetcode/duplicatesinarray.cpp [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
+        return ans;
     }
-    if(ans.size()==0){
-        return {-1};
-    }
-    else return ans;
-}
+};
 
-
```
Comment (Saikat2407) [inline Leetcode/duplicatesinarray.cpp]: Add the main fuction

File: Leetcode/duplicatesinarray.cpp [hunk 3]
```diff
@@ -1,31 +1,18 @@
...
+            }
         }
+        return ans;
     }
-    if(ans.size()==0){
-        return {-1};
-    }
-    else return ans;
-}
+};
```
Comment (Saikat2407) [inline Leetcode/duplicatesinarray.cpp:17]: 1. remove ";" from line 17

**Review by Saikat2407 — CHANGES_REQUESTED** [review]

**Comment by Abulaman23** [comment]
Changes done,do look into it!

**Review by Saikat2407 — APPROVED** [review]
**Approved**

**Pull request merged**